### ISSUE TITLE
No project manager in Pydriller

### ISSUE THREAD

**Issue by TylerPham2000** [GatorCogitate/cogitate_tool#8]
Every other research group has a project manager but our research group has no project manager. Not sure if we need one or not.

**Comment by guzmanr04** [comment]
I have also noticed the same issue.

**Comment by koscinskic** [comment]
I don't think the research facet of the team structure needs a project manager, but I'll familiarize myself with the Pydriller documentation just in case there's a need for it. I'll bring it up tomorrow so the issue gets sorted out before the lab.

**Comment by munzekm** [comment]
Thank you for addressing this issue and handling it tomorrow in lab. That way it will clear up any confusion.

**Comment by declanjcasey** [comment]
Hey Megan, what features did you add last night?

**Comment by koscinskic** [comment]
After meeting with the other project managers, we've determined that the research teams were temporary whereas the feature / task teams will be better developed to support operations for the rest of the project's duration. Consequentially, we did not find that the lack of a project manager with knowledge of Pydriller is a significant problem. However, If there are questions concerning the tool, feel free to ask me and I'll look into it.
